platforms/qhaikuplatform/haiku/qhaikuplatformfontdatabase.cpp

```diff
@@ -129,23 +138,34 @@ QStringList	QHaikuPlatformFontDatabase::fallbacksForFamily(const QString &family
 	QStringList result;
 	if (styleHint == QFont::Monospace || styleHint	== QFont::Courier) {
 		result.append(QString::fromLatin1("Noto Sans Mono"));
-		result.append(QString::fromLatin1("DejaVu	Sans Mono"));
+		result.append(QString::fromLatin1("Noto Sans CJK JP"));
+		result.append(QString::fromLatin1("Noto Sans Thai"));
+		result.append(QString::fromLatin1("DejaVu Sans Mono"));
 	} else {
 		if (styleHint	== QFont::Serif) {
 			result.append(QString::fromLatin1("Noto Serif"));
+			result.append(QString::fromLatin1("Noto Serif CJK JP"));
+			result.append(QString::fromLatin1("Noto Serif Thai"));
 			result.append(QString::fromLatin1("DejaVu Serif"));
 		} else {
-			result.append(QString::fromLatin1("Noto Sans	Display"));
+			result.append(QString::fromLatin1("Noto Sans"));
+			result.append(QString::fromLatin1("Noto Sans CJK JP"));
+			result.append(QString::fromLatin1("Noto Sans Thai"));
 			result.append(QString::fromLatin1("DejaVu Sans"));
 		}
 	}
+
+	result.append(QString::fromLatin1("Noto Sans Symbols"));
+	result.append(QString::fromLatin1("Noto Sans Symbols 2"));
+	result.append(QString::fromLatin1("Noto Sans Emoji"));
+
 	result.append(QFreeTypeFontDatabase::fallbacksForFamily(family, style,	styleHint, script));
 	return	result;
 }
 
 QFont QHaikuPlatformFontDatabase::defaultFont()	const
 {
-	QFont font(QStringLiteral("Noto Sans Display"));
+	QFont font(QStringLiteral("Noto Sans"));
```

Comment:
And here, I guess?

Reply:
(Or, just return "Sans Serif" here?)

Reply:
I didn't think of it. Yep, will change for be_plain_font.